docs/core/install/localized-intellisense.md

```diff
@@ -1,13 +1,11 @@
 ---
 title: Install localized IntelliSense files
 description: Learn how to set up your development machine to use localized IntelliSense files for .NET Core projects in Visual Studio.
-author: mairaw
-ms.author: mairaw
 ms.date: 12/18/2019
 ---
 # How to install localized IntelliSense files for .NET Core
 
-[IntelliSense](/visualstudio/ide/using-intellisense) is a code-completion aid that is available in different integrated development environments (IDEs), such as Visual Studio. By default, when you're developing .NET Core projects, the SDK only includes the English version of the IntelliSense files. This article explains:
+[IntelliSense](/visualstudio/ide/using-intellisense) is a code-completion aid that is available in different Integrated Development Environments (IDEs), such as Visual Studio. By default, when you're developing .NET Core projects, the SDK only includes the English version of the IntelliSense files. This article explains:
```

Comment:
Better to use it capitalized, as it has an acrynom ?
Feel free if you want to revert this change.

Reply:
Our internal style guide says "IDE - Acronym for integrated development environment" so I think it was correct before

Reply:
@mairaw, Okay I'll revert that one.